rero_ils/modules/serializers/base.py

```diff
@@ -37,14 +40,22 @@ class JSONSerializer(_JSONSerializer, PostprocessorMixin):
 
     def preprocess_record(self, pid, record, links_factory=None, **kwargs):
         """Prepare a record and persistent identifier for serialization."""
-        rec = record
+        links_factory = links_factory or (lambda x, record=None, **k: dict())
```

Comment:
you should use shorter lambda : `lambda *a, **k: dict()`

Reply:
I would do touch this as it is a copy/paste from invenio.